src/hotspot/os/windows/memMapPrinter_windows.cpp

```diff
@@ -175,7 +175,7 @@ class MappingInfoPrinter {
     INDENT_BY(57);
     st->print("%s-%s", mapping_info._state_buffer.base(), mapping_info._type_buffer.base());
     INDENT_BY(63);
-    st->print("%#11llx", reinterpret_cast<const unsigned long long>(mem_info.BaseAddress) - reinterpret_cast<const unsigned long long>(mem_info.AllocationBase));
+    st->print("%#11llx", reinterpret_cast<unsigned long long>(mem_info.BaseAddress) - reinterpret_cast<unsigned long long>(mem_info.AllocationBase));
```

Comment:
What is the issue with const here?

Reply:
More a correctness issue. The const is not required since the cast result is an rvalue unsigned long long. It's similar to something like a method's return type being marked as `const int get();` where the const doesn't really mean anything since there is no possible way for it to be modified